README.md

```diff
@@ -51,6 +51,63 @@ This repository includes the following:
 - References to the course videos.
 - Supplementary slides for various topics covered in the course.
 
+## Event
```

Comment:
Please reference to:  https://meeting.tencent.com/v2/cloud-record/share?id=69c7740d-2cdc-4475-9985-2edf4345e54c&from=3&is-single=false&record_type=2

And rewrite this session.